src/components/popups/genericrequestpopup.vue

```diff
@@ -64,7 +68,7 @@
                     );
                     let transaction = await blockchain.sign(
                         operation,
-                        this.incoming.signingAccount.keys.active
+                        await getKey(this.$store.getters['AccountStore/getSigningKey'](this.incoming).keys.active)
```

Comment:
Should we send the whole incoming object, or filter out the required public key / account id?

Reply:
what do you mean? we're not really sending anything...it's just a getter, it needs accountID + chain (and in the future keytype or something) so we can just pass the incoming object or create a new one with the required props and send that... Since it's just a reference, I prefer the first option.